lightning/src/ln/onion_utils.rs

```diff
@@ -367,7 +367,7 @@ pub(super) fn process_onion_failure<T: secp256k1::Signing, L: Deref>(secp_ctx: &
 						const NODE: u16 = 0x2000;
 						const UPDATE: u16 = 0x1000;
 
-						let error_code = byte_utils::slice_to_be16(&error_code_slice);
+						let error_code = u16::from_be_bytes(error_code_slice.try_into().expect("len is 2"));
```

Comment:
nit: the message will prefix the error, so maybe something specific to the call site instead for these.

Reply:
I sure hope we never *see* it anywhere, so I'll just leave it :).